### PR TITLE
Github Actions workflow definition

### PR DESCRIPTION
**Pull request by NadeeshaGarusinghe**

### CODE REVIEW

File: .github/workflows/learn-github-actions.yml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 name: learn-github-actions
 run-name: ${{ github.actor }} is learning GitHub Actions
-on: [push]
+on:
```
Comment (sadilchamishka) [inline .github/workflows/learn-github-actions.yml:3]: Add another job

File: .github/workflows/learn-github-actions.yml [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
 jobs:
     check-bats-version:
         runs-on: ubuntu-latest
```
Comment (sadilchamishka) [inline .github/workflows/learn-github-actions.yml:12]: Add new line

File: .github/workflows/learn-github-actions.yml (hunk 1 above)
Comment (NadeeshaGarusinghe) [reply .github/workflows/learn-github-actions.yml:3]: done!

File: .github/workflows/learn-github-actions.yml (hunk 2 above)
Comment (NadeeshaGarusinghe) [reply .github/workflows/learn-github-actions.yml:12]: done!

**Review by sadilchamishka — APPROVED** [review]

**Pull request merged**